modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/BuilderScreen/BuilderListItem.scss

```diff
@@ -21,24 +21,14 @@
 		margin-right: 10px;
 	}
 
-	&__object-builder-list-item-first-column--not-draggable {
-		margin-left: 10px;
-	}
-
 	&__object-builder-list-item-second-column {
-		left: 336px;
+		left: 336px !important;
 		position: absolute;
 		top: 23px;
-
-		&--not-draggable {
-			left: 415px !important;
-			position: absolute;
-			top: 23px;
-		}
 	}
 
-	&__object-builder-list-item-third-column--not-draggable {
-		left: 785px !important;
+	&__object-builder-list-item-third-column.not-draggable {
```

Comment:
@danvb112 Thanks for pointing it out. I was trying to avoid duplicating code in the css, but now I see that I could have done this without changing the nomination :)